modules/navier_stokes/include/physics/WCNSFVCoupledAdvectionPhysicsHelper.h

```diff
@@ -31,6 +31,9 @@ class WCNSFVCoupledAdvectionPhysicsHelper
   /// It is important to forward to the Physics so we do not get the smoothing status wrong
   MooseFunctorName getPorosityFunctorName(bool smoothed) const;
 
+  MooseFunctorName densityName() const { return _density_name; }
+  MooseFunctorName dynamicViscosityName() const { return _dynamic_viscosity_name; }
```

Comment:
these could return const reference to avoid string copies

Reply:
changed